src/clients/api/queries/getPendingRewards/formatOutput/formatRewardSummaryData.ts

```diff
@@ -52,9 +61,25 @@ function formatRewardSummaryData(rewardSummary: RewardSummary): FormatRewardSumm
     return;
   }
 
+  const rewardTokenPrice = rewardTokenPrices[rewardToken.address.toLowerCase()];
+  const rewardTokenPriceCents = convertDollarsToCents(rewardTokenPrice);
+
+  // return if there is no available reward token price
+  if (!rewardTokenPrice) {
+    return;
+  }
```

Comment:
I think in this instance we should make the `rewardAmountCents` optional and only display the corresponding price of a reward in dollars next to its value in tokens if that prop was defined. This way we'd still be able to display rewards even if a price could not be fetched for it and so users would still be able to claim it.
What do you think?